mlos_bench/mlos_bench/tests/environments/local_env_test.py

```diff
@@ -66,3 +66,30 @@ def test_local_env(tunable_groups: TunableGroups) -> None:
             (ts2, "cpu_load", 0.8),
             (ts2, "mem_usage", 20480.0),
         ]
+
+
+def test_local_env_wide(tunable_groups: TunableGroups) -> None:
+    """
+    Produce benchmark data in wide format and read it.
+    """
+    local_env = LocalEnv(
+        name="Test Local Env",
+        config={
+            "run": [
+                "echo 'latency,throughput,score' > output.csv",
```

Comment:
Can we add one more that doesn't include the header?
And also a negative test that has the wrong number of columns or wrong format of the columns?